types/module/server/manager.go

```diff
@@ -156,6 +171,10 @@ func (c *configurator) QueryServer() gogogrpc.Server {
 	return c.queryServer
 }
 
+func (c *configurator) RegisterInvariants(registry RegisterInvariantsHandler) {
```

Comment:
```suggestion
func (c *configurator) RegisterInvariantsHandler(registry RegisterInvariantsHandler) {
```

Reply:
We should not change RegisterInvariants since this is a part of AppModule interface

Reply:
I don't get it, `configurator` doesn't implement `AppModule` interface